tools/autodeployment/deploy.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+if [ $1 = 'production']
+then
+  DOCKER_FILE=docker-compose-production.yml
+else
+  DOCKER_FILE=docker-compose-staging.yml
+fi
+
+# Shutdown
+cd ../telescope
+
+docker-compose -f $DOCKER_FILE down
+
+docker system prune -af --volumes
+
+
+# Delete and clone only if the first argument is staging
+if [ $1 = 'staging' ]
+then
+  cd ..
+
+  rm -rf telescope
+
+  git clone https://github.com/Seneca-CDOT/telescope.git --depth=1
+fi
+
+
+# Deploy
+cd telescope/
+
+# Takes the second argument the name of the env file to be used
+cp $2 .env
```

Comment:
We should maybe add a check to make sure that $2 is defined and is an existing file.

```bash
if [[ -f $2 ]]; then
...
```

Reply:
Going to forego the second argument and set it in the script itself, if that is alright